test/lib/PlacesTestUtils.js

```diff
@@ -11,6 +12,7 @@ XPCOMUtils.defineLazyModuleGetter(this, "PlacesUtils",
   "resource://gre/modules/PlacesUtils.jsm");
 
 const PlacesTestUtils = Object.freeze({
```

Comment:
This is a straight port of https://dxr.mozilla.org/mozilla-central/rev/2dfb45d74f42d2a0010696f5fd47c7a7da94cedb/toolkit/components/places/tests/PlacesTestUtils.jsm

was done in #58